src/sentry/integrations/pagerduty/integration.py

```diff
@@ -0,0 +1,185 @@
+from __future__ import absolute_import
+
+from django.utils.translation import ugettext_lazy as _
+from django.db import transaction
+
+from sentry import options
+
+from sentry.utils import json
+from sentry.integrations import (
+    IntegrationInstallation,
+    IntegrationFeatures,
+    IntegrationMetadata,
+    IntegrationProvider,
+    FeatureDescription,
+)
+from sentry.models import OrganizationIntegration, PagerDutyServiceProject, PagerDutyService, Project
+from sentry.pipeline import PipelineView
+
+DESCRIPTION = """
+PagerDuty Description
+"""
+
+FEATURES = [
+    FeatureDescription(
+        """
+        Configure rule based PagerDuty notifications!!
+        """,
+        IntegrationFeatures.ALERT_RULE,
+    )
+]
+
+metadata = IntegrationMetadata(
+    description=_(DESCRIPTION.strip()),
+    features=FEATURES,
+    author="The Sentry Team",
+    noun=_("Installation"),
+    issue_url="https://github.com/getsentry/sentry/issues/new?title=PagerDuty%20Integration:%20&labels=Component%3A%20Integrations",
+    source_url="https://github.com/getsentry/sentry/tree/master/src/sentry/integrations/pagerduty",
+    aspects={},
+)
+
+
+class PagerDutyIntegration(IntegrationInstallation):
+    def get_client(self):
+        pass
```

Comment:
Seems incomplete.

Reply:
so I've added `return PagerDutyClient(integration_key=integration_key)` but I'm not sure how weird that is given that it's not really how any of the other integrations work. But we don't have like a single auth token or anything for the client like we do the others